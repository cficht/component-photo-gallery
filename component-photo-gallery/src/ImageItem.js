import React, { Component } from 'react';

export default class ImageItem extends Component {
    render() {
        return (
                <li className="list-item">
                    <h2>{this.props.horned.title}</h2>
                    <img src={this.props.horned.url} alt=""></img>
                    <p>{this.props.horned.description}</p>
                    <h5>Keyword: {this.props.horned.keyword}</h5>
                    <h5>Horns: {this.props.horned.horns}</h5>
                </li>
        );
    }
}