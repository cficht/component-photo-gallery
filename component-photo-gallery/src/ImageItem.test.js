import React from 'react';
import { render } from '@testing-library/react';
import images from './data.js';
import ImageItem from './ImageItem.js';
import renderer from "react-test-renderer";

test('renders ImageItem.js correctly', () => {
  const tree = renderer
  .create(<ImageItem horned={images} />)
  .toJSON();
  expect(tree).toMatchSnapshot();
});