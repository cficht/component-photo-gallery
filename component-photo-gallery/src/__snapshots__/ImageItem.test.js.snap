// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ImageItem.js correctly 1`] = `
<li
  className="list-item"
>
  <h2 />
  <img
    alt=""
  />
  <p />
  <h5>
    Keyword: 
  </h5>
  <h5>
    Horns: 
  </h5>
</li>
`;
